src/site/content/en/blog/aria-poison-or-antidote/index.md

```diff
@@ -219,16 +219,6 @@ they're considered shippable. There's too much nuance. Ideally everything would
 several different browser-screen reader combinations, because of the numerous implementation
 quirks, in addition to a few incomplete implementations.
 
-### File bug on Chrome or on web content?
-
-For our team, which works on Chrome, we need to know when our representation of truth doesn't match
-what's provided by other browsers. When NVDA reads something different in the same web content in
-Firefox vs. Chrome, it's often our bug, since Firefox has an older, more seasoned accessibility
-mapping layer.
-
-If all the browser-screen reader combinations seem to agree, and the web content still behaves
-badly, we will need to file a bug on the web content itself, not on the browser.
-
```

Comment:
We ask for feedback on many of our articles. Why do you want to remove this?

Reply:
Joe & I synced about this in Chat